MozillaFoundation/foundation.mozilla.org
# updated author_card.html to have fallback image

## Description
Related PRs/issues: #10326
Links to sample test pages: [Research Authors Index page](https://foundation-s-10326-rcc--7xjbpi.herokuapp.com/en/research/authors/), [Research Detail Page](https://foundation-s-10326-rcc--7xjbpi.herokuapp.com/en/research/library/from-real-look-toward/)


**This PR includes the following updates:**
- Updates the template `author_card.html` to use the default grey/black wave placeholder image, if an author does not have a profile picture set.  
- Since we are now using this placeholder image in the research hub / RCC, I have updated the placeholder image's file name from `blog-author-placeholder.jpg` to `author-placeholder.jpg`.
- Updates existing templates that use the placeholder image to account for the name change.


## Steps to test:

1.  Visit the review app's [Research Detail Page](https://foundation-s-10326-rcc--7xjbpi.herokuapp.com/en/research/library/from-real-look-toward/)
2. Note that the author "Sharon Wright" does not have a profile image set (placeholder image), while the other related profiles are using their assigned images (blue).
3. Visit the review app's [Research Authors Index page](https://foundation-s-10326-rcc--7xjbpi.herokuapp.com/en/research/authors/)
4.  Note that the author "Sharon Wright" does not have a profile image set (placeholder image), while the other related profiles are using their assigned images (blue).
5. If everything is looking as expected, testing is complete!
